cmd/deploy/cfghandler.go

```diff
@@ -72,7 +71,6 @@ func (ch *defaultConfigMapHandler) updateConfigMap(ctx context.Context, cfg *api
 		return err
 	}
 	if errMain != nil {
-		oktetoLog.AddToBuffer(oktetoLog.ErrorLevel, errMain.Error())
```

Comment:
log buffer already has this error added. If we add again the error we will see it duplicate